docs/misc/changelog.rst

```diff
@@ -10,6 +10,8 @@ Pre-Release 2.5.1a0 (WIP)
 
 - doc update (fix example of result plotter + improve doc)
 - fixed logger issues when stdout lacks ``read`` function
+- clarified the public interface of ``BasePolicy`` and ``ActorCriticPolicy``. Breaking change: `masks_ph` is now called `dones_ph`.
```

Comment:
```suggestion
- clarified the public interface of ``BasePolicy`` and ``ActorCriticPolicy``. **Breaking change** when using custom policies: ``masks_ph`` is now called ``dones_ph``.
```

minor: formatting + highlight breaking change

Reply:
maybe also (but this is not public api), `_value` renamed `value_flat`

Reply:
Maybe we should somewhere clearly define what "public api" actually is?